docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -923,6 +942,29 @@ logic. In this example, both the event's `destination.ip` and
 <4> Sibling `entries` are evaluated using `or` logic. An alert is generated when
 at least one `entries` object evaluates to `true`.
 
+*Example 6*
+
+New terms rule that creates alerts when the Windows `rundll32.exe` process makes
+unusual network connections:
```

Comment:
Just checking if this is correct as it doesn't seem to correspond to what the example is showing?